src/current/cockroachcloud/troubleshooting-page.md

```diff
@@ -164,10 +164,10 @@ Using `sslmode=require` can leave your cluster vulnerable to MITM and impersonat
 
 ### Hanging or stuck queries
 
-When you experience a hanging or stuck query using {{ site.data.products.serverless }}, the cause could be that you've reached your [resource limit](learn-about-pricing.html#choosing-resource-limits) for [Request Units](learn-about-request-units.html) or storage space. This can affect SQL Statements or `cockroach` CLI commands. To check your cluster's status, navigate to the [**Cluster Overview** page](cluster-overview-page.html) in the Cloud Console. In the **Usage this month** section, you will see your resource usage and limits. If you've used all your storage, your cluster will be labeled **THROTTLED**, and you will only be able to delete data. If you've used all your RUs, your cluster will be **DISABLED**.
+When you experience a hanging or stuck query using {{ site.data.products.serverless }}, the cause could be that you've reached your [resource limit](plan-your-cluster-serverless.html#choosing-resource-limits) for [Request Units](plan-your-cluster-serverless.html#request-units) or storage space. This can affect SQL Statements or `cockroach` CLI commands. To check your cluster's status, navigate to the [**Cluster Overview** page](cluster-overview-page.html) in the Cloud Console. In the **Usage this month** section, you will see your resource usage and limits. If you've used all your storage, your cluster will be labeled **THROTTLED**, and you will only be able to delete data. If you've used all your RUs, your cluster will be **DISABLED**.
```

Comment:
We need to remove the "hanging or stuck query" language, since we're no longer throttling individual queries when running out of resources. The "throttling" that we do now only happens when the storage limit is breached, and that consists of restricting the user to a single SQL connection query which the user can use to delete data. We have no way to ensure that they only can delete data, however.